### PR TITLE
Add Control Plane Config for AWS in expand_spec.go

### PR DESCRIPTION
**Pull request by neo-liang-sap**

### CODE REVIEW

**Comment by neo-liang-sap** [comment]
seems there's some issue with `license/cla` checks - i have signed it before but now it keeps pending?

File: expand/expand_spec.go [hunk 1]
```diff
@@ -309,6 +310,15 @@ func getAzControlPlaneConfig() *corev1beta1.ProviderConfig {
 	return &obj
 }
 
+func getAwsControlPlaneConfig() *corev1beta1.ProviderConfig {
```
Comment (rakesh-garimella) [inline expand/expand_spec.go]: small change please remove getXXXX. Make it only XXXX. As mentioned here: https://golang.org/doc/effective_go.html#Getters

Comment (neo-liang-sap) [reply expand/expand_spec.go]: thanks! i've changed function name and function reference

**Review by rakesh-garimella — APPROVED** [review]

**Comment by neo-liang-sap** [comment]
thanks! @rakesh-garimella

**Pull request merged**